notification-service/src/main/java/org/hypertrace/alert/engine/notification/service/notification/MetricAnomalySlackEvent.java

```diff
@@ -34,6 +38,10 @@ static MetricAnomalySlackEvent getMessage(MetricAnomalyWebhookEvent metricAnomal
         metadataFields, metricAnomalyWebhookEvent.getViolationTimestamp(), VIOLATION_TIMESTAMP);
     addIfNotEmpty(
         metadataFields, metricAnomalyWebhookEvent.getEventConditionId(), METRIC_ANOMALY_EVENT_TYPE);
+    addIfNotEmpty(
+        metadataFields,
+        getMetricValues(metricAnomalyWebhookEvent.getMetricValuesList()),
```

Comment:
As suggested earlier lets drop the individual metric values and change the message to 
``` `x of y` metric data point were greater than the threshold `z` ```
where x is number of violated data points and y is total data point and z is the threshold

Reply:
And change the message key to `Violation Summary`

Reply:
Bur aren't we returning a violation only when all data points are violated ?